### PR TITLE
Add a check for memory unit

### PR DESCRIPTION
**Pull request by shruacha1234**
Signed-off-by: Shruthi <Shruthi.Acharya@ibm.com>

### CODE REVIEW

**Review by dinogun — CHANGES_REQUESTED** [review]
We cannot assume that it is always in `MiB` units. We need to add a comment in the "command help" section that this unit needs to be always specified in `MiB`

**Comment by shruacha1234** [comment]
> We cannot assume that it is always in `MiB` units. We need to add a comment in the "command help" section that this unit needs to be always specified in `MiB`

Need not add a check for memory unit?

File: spring-petclinic/scripts/petclinic-deploy-openshift.sh [hunk 1]
```diff
@@ -87,6 +89,46 @@ if [ -z "${NAMESPACE}" ]; then
 	NAMESPACE="${DEFAULT_NAMESPACE}"
 fi
 
+# Check if the memory request/limit has unit. If not ask user to append the unit
+case "${MEM_REQ}" in
```
Comment (dinogun) [inline spring-petclinic/scripts/petclinic-deploy-openshift.sh:93]: Suggest you create a function for checking the memory unit and call it for both `MEM_REQ` and `CPU_REQ`.

**Review by dinogun — APPROVED** [review]
lgtm !

**Pull request merged**